go/tosca/vm/opcodes_test.go

```diff
@@ -81,28 +79,12 @@ func TestOpCode_NumberOfOpCodes(t *testing.T) {
 		CREATE, CALL, CALLCODE, RETURN, DELEGATECALL, CREATE2, STATICCALL, REVERT, INVALID, SELFDESTRUCT,
 	}
 
-	if NewestSupportedRevision > tosca.R13_Cancun {
-		t.Errorf("Missing update for revision %v", NewestSupportedRevision)
-	}
-
 	currentOpCodes := istanbulOpCodes
-	switch NewestSupportedRevision {
-	case tosca.R13_Cancun:
-		// TODO: enable once supported
-		// currentOpCodes = append(currentOpCodes, []OpCode{BLOBHASH, BLOBBASEFEE, TLOAD, TSTORE, MCOPY}...) // Cancun
-		fallthrough
-	case tosca.R12_Shanghai:
-		currentOpCodes = append(currentOpCodes, []OpCode{PUSH0}...) // Shanghai
-		fallthrough
-	case tosca.R11_Paris:
-		currentOpCodes = append(currentOpCodes, []OpCode{}...) // Paris
-		fallthrough
-	case tosca.R10_London:
-		currentOpCodes = append(currentOpCodes, []OpCode{BASEFEE}...) // London
-		fallthrough
-	case tosca.R09_Berlin:
-		currentOpCodes = append(currentOpCodes, []OpCode{}...) // Berlin
-	}
+	currentOpCodes = append(currentOpCodes, []OpCode{}...)                                            // Berlin
```

Comment:
As discussed, I am not sure about the use case of this unit test. But I would leave it as it was.

Reply:
I do not have strong opinions on the matter. I also don't think leaving a `switch` with all `fallthrought` that will pass, does not make much sense either.

Reply:
This unit test was added before the new revision updates were applied. I think the switch case can and should be removed now that we have full support.